frontend/src/components/layout/header/HeaderView.jsx

```diff
@@ -1,8 +1,29 @@
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Avatar from '@mui/material/Avatar';
+import logo from '../../../images/logo.png'
 import classes from './header.module.scss'
 
-const HeaderView = () => (
+const HeaderView = ({goToFeed, goToProfile}) => (
     <div className={classes.container}>
-        Header
+        <Box sx={{ flexGrow: 1 }}>
+            <AppBar position="static">
+                <Toolbar>
+                    <IconButton onClick={goToFeed} sx={{ mr: 2 }}>
+                        <img alt="logo" className={classes.photo} src={logo} />
+                    </IconButton>
+                    <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+                        Updog
+                    </Typography>
+                    <IconButton onClick={goToProfile}>
+                        <Avatar alt="Barrack Obama" src="https://i.imgur.com/PiJAoqO.jpeg" />
```

Comment:
You should load some mock data in the controller and pass it in as a prop to the view to use as the alt and src instead of hard coding it here. For reference you can see how it is done on the Post Page. There is some data to use in #39